docs/markdown/overview.md

```diff
@@ -0,0 +1,67 @@
+# Interledger Overview
+
+Interledger enables the seamless exchange of value across different payment networks. Traditional payment networks 
+operate independently from each other. Sending value is easy only if the sender and recipient have accounts on the same 
+network, but it can be slow and expensive if they have accounts on different networks. Interledger makes it easy to 
+transact in whatever currency or payment network you choose because it is not tied to any one company, blockchain, or 
+currency. Using Interledger, you can easily send XRP to someone who wants to receive ETH, or you can send USD to someone 
+who wants to receive EUR.
+
+
+## What is Interledger?
+Interledger is a network of computers that enables the sending of value across independent payment networks. 
+Similar to how the Internet routes packets information, Interledger routes packets of value. Computers on the Interledger 
+network are called *nodes*. Nodes can take one or more of the following roles: 
+* Sender – Initiates a value transfer.
+* Router – Applies currency exchange and forwards packets of value. This is an intermediary node between the sender 
+and the receiver. 
+* Receiver – Receives the value.
+
+![ILP-nodes](../images/ilp-nodes.png)
+
+>Note: The terms *Router* and *Connector* are used interchangeably throughout the documentation.
```

Comment:
If they're used interchangeably we should stick to one for clarity

Reply:
I agree with you and that's the eventual goal. But we haven't settled on that one word yet, and currently, in the docs, we have links to RFCs (under Specs in the left nav) that use the word `connector` everywhere. Once we settle on the term - router or connector- and have more content on the website so that we can remove the links to RFCs(from the left nav), we can remove this statement too.